Content.Server/Nyanotrasen/Psionics/Glimmer/PassiveGlimmerReductionSystem.cs

```diff
@@ -25,7 +29,7 @@ public override void Update(float frameTime)
                 return;
 
             var delta = curTime - LastUpdateTime;
-            var maxGlimmerLost = (int) Math.Round(delta.TotalSeconds * GlimmerLostPerSecond);
+            var maxGlimmerLost = (int) Math.Round(delta.TotalSeconds * _cfg.GetCVar(CCVars.GlimmerLostPerSecond));
```

Comment:
Since this is run so frequently, it may be more performant to cache this value (`GlimmerLostPerSecond`) when the cvar is changed and reference that instead. There are examples of this being done in ExplosionSystem and AtmosphereSystem, see `OnValueChanged` for examples.